Slim/Plugin/Podcast/Parser.pm

```diff
@@ -50,14 +52,8 @@ sub parse {
 		# track progress of our listening
 		my $key = 'podcast-' . $item->{enclosure}->{url};
 		my $position = $cache->get($key);
-		if ( !$position ) {
-			if ( my $redirect = $cache->get("$key-redirect") ) {
-				$position = $cache->get("podcast-$redirect");
-			}
-
-			$cache->set($key, $position || 0, '30days');
-		}
-
+		$cache->set($key, $position || 0, '30days') unless $position;
```

Comment:
This would always set the value to 0, as the value is only set if `$position` was falsy.

Reply:
indeed :-)